erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -233,9 +233,21 @@ def set_liability_account(self):
 				self.is_opening = "No"
 				return
 
-		liability_account = get_party_account(
-			self.party_type, self.party, self.company, include_advance=True
-		)[1]
+		accounts = get_party_account(self.party_type, self.party, self.company, include_advance=True)
+
+		liability_account = accounts[1] if len(accounts) > 1 else None
+		fieldname = (
```

Comment:
`fieldname` is unused. Were you intending to use it somewhere?

Reply:
It is used in error message.